parser/src/test/scala/hmda/parser/fi/ts/TsDatParserSpec.scala

```diff
@@ -19,34 +21,43 @@ class TsDatParserSpec extends FlatSpec with MustMatchers {
     respondent.zipCode mustBe "22102"
   }
 
-  it should "parse other info about the respondent" in {
+  property("generate proper dat format") {
+    forAll(tsGen) { (ts: TransmittalSheet) =>
+      val parsedTs = TsDatParser(ts.toDAT)
+      val updatedRespondent = parsedTs.respondent.copy(id = parsedTs.respondent.id.replaceFirst("^0+(?!$)", ""))
```

Comment:
So, currently when the parser persists the Respondent ID, it keeps all the unnecessary 0s at the front? That's a bug in the parser, right? 

Sounds like we should make a ticket to make the DAT parser work as the specification states.